--- swift-corelibs-libdispatch-swift-5.3.3-RELEASE/CMakeLists.txt	2020-09-02 07:28:48.000000000 +0000
+++ ./CMakeLists.txt	2021-02-10 01:20:11.108747828 +0000
@@ -62,7 +62,7 @@
 # NOTE(abdulras) this is the CMake supported way to control whether we generate
 # shared or static libraries.  This impacts the behaviour of `add_library` in
 # what type of library it generates.
-option(BUILD_SHARED_LIBS "build shared libraries" ON)
+option(BUILD_SHARED_LIBS "build shared libraries" OFF)
 
 option(DISPATCH_ENABLE_ASSERTS "enable debug assertions" FALSE)
 
