--- bat-0.15.2/src/bin/bat/clap_app.rs	2020-05-25 15:27:27.000000000 +0000
+++ ./src/bin/bat/clap_app.rs	2020-05-26 09:45:16.334866102 +0000
@@ -1,4 +1,4 @@
-use clap::{crate_name, crate_version, App as ClapApp, AppSettings, Arg, ArgGroup, SubCommand};
+use clap::{crate_name, App as ClapApp, AppSettings, Arg, ArgGroup, SubCommand};
 use std::path::Path;
 
 pub fn build_app(interactive_output: bool) -> ClapApp<'static, 'static> {
@@ -9,7 +9,7 @@
     };
 
     let mut app = ClapApp::new(crate_name!())
-        .version(crate_version!())
+        .version("@MAGISK_MODULE_VERSION@ - Compiled for Android by nelshh @ XDA")
         .global_setting(clap_color_setting)
         .global_setting(AppSettings::DeriveDisplayOrder)
         .global_setting(AppSettings::UnifiedHelpMessage)
