--- bat-0.16.0/src/bin/bat/clap_app.rs	2020-10-02 21:27:29.000000000 +0000
+++ ./src/bin/bat/clap_app.rs	2020-10-03 01:03:50.426111678 +0000
@@ -1,4 +1,4 @@
-use clap::{crate_name, crate_version, App as ClapApp, AppSettings, Arg, ArgGroup, SubCommand};
+use clap::{crate_name, App as ClapApp, AppSettings, Arg, ArgGroup, SubCommand};
 use std::env;
 use std::path::Path;
 
@@ -10,7 +10,7 @@
     };
 
     let mut app = ClapApp::new(crate_name!())
-        .version(crate_version!())
+        .version("@MAGISK_MODULE_VERSION@ - Compiled for Android by nelshh @ XDA")
         .global_setting(clap_color_setting)
         .global_setting(AppSettings::DeriveDisplayOrder)
         .global_setting(AppSettings::UnifiedHelpMessage)
