--- ../src/clap_app.rs	2019-07-07 13:05:02.590686985 +0200
+++ src/clap_app.rs	2019-07-07 13:06:12.392537180 +0200
@@ -13,7 +13,7 @@
     let arg_group_required = !Path::new("cache").exists();
 
     ClapApp::new(crate_name!())
-        .version(crate_version!())
+        .version("0.11.0 - Compiled for Android by nelshh @ XDA")
         .global_setting(clap_color_setting)
         .global_setting(AppSettings::DeriveDisplayOrder)
         .global_setting(AppSettings::UnifiedHelpMessage)
