--- Makefile.linux-gcc.old	2020-10-30 20:14:37.280296716 +0000
+++ ./Makefile.linux-gcc	2020-10-30 20:14:46.612577664 +0000
@@ -14,7 +14,7 @@
 #### The toplevel directory of the source tree.  This is the directory
 #    that contains this "Makefile.in" and the "configure.in" script.
 #
-TOP = ../sqlite
+TOP = .
 
 #### C Compiler and options for use in building executables that
 #    will run on the platform that is doing the build.
