--- ../unix/configure	2008-06-20 03:32:20.000000000 +0000
+++ ./unix/configure	2019-07-16 11:11:43.606537287 +0000
@@ -19,7 +19,7 @@
 CC=${1-cc}
 CFLAGS=${2-"-I. -DUNIX"}
 LFLAGS1=''
-LFLAGS2=''
+LFLAGS2=' --static'
 LN="ln -s"
 
 CFLAGS_OPT=''
@@ -382,8 +382,7 @@
   CFLAGS="${CFLAGS} -DUIDGID_NOT_16BIT"
 else
 # run it
-  ./conftest
-  r=$?
+  r=3
   if [ $r -eq 1 ]; then
     echo -- UID not 2 bytes - disabling old 16-bit UID/GID support
     CFLAGS="${CFLAGS} -DUIDGID_NOT_16BIT"
@@ -435,15 +434,14 @@
   echo -- no Large File Support
 else
 # run it
-  ./conftest
-  r=$?
+  r=3
   if [ $r -eq 1 ]; then
     echo -- no Large File Support - no 64-bit off_t
   elif [ $r -eq 2 ]; then
     echo -- no Large File Support - no 64-bit stat
   elif [ $r -eq 3 ]; then
     echo -- yes we have Large File Support!
-    CFLAGS="${CFLAGS} -DLARGE_FILE_SUPPORT"
+    CFLAGS="${CFLAGS} -DLARGE_FILE_SUPPORT -DZIP64_SUPPORT"
   else
     echo -- no Large File Support - conftest returned $r
   fi
@@ -509,83 +507,6 @@
 # Check for missing functions
 # add NO_'function_name' to flags if missing
 
-for func in rmdir strchr strrchr rename mktemp mktime mkstemp
-do
-  echo Check for $func
-  echo "int main(){ $func(); return 0; }" > conftest.c
-  $CC $BFLAG -o conftest conftest.c >/dev/null 2>/dev/null
-  [ $? -ne 0 ] && CFLAGS="${CFLAGS} -DNO_`echo $func | tr '[a-z]' '[A-Z]'`"
-done
-
-
-echo Check for memset
-echo "int main(){ char k; memset(&k,0,0); return 0; }" > conftest.c
-$CC -o conftest conftest.c >/dev/null 2>/dev/null
-[ $? -ne 0 ] && CFLAGS="${CFLAGS} -DZMEM"
-
-
-echo Check for memmove
-cat > conftest.c << _EOF_
-#include <string.h>
-int main() { int a; int b = 0; memmove( &a, &b, sizeof( a)); return a; }
-_EOF_
-$CC -o conftest conftest.c >/dev/null 2>/dev/null
-[ $? -ne 0 ] && CFLAGS="${CFLAGS} -DNEED_MEMMOVE"
-
-
-echo Check for strerror
-cat > conftest.c << _EOF_
-#include <string.h>
-int main() { strerror( 0); return 0; }
-_EOF_
-$CC -o conftest conftest.c >/dev/null 2>/dev/null
-[ $? -ne 0 ] && CFLAGS="${CFLAGS} -DNEED_STRERROR"
-
-echo Check for errno declaration
-cat > conftest.c << _EOF_
-#include <errno.h>
-main()
-{
-  errno = 0;
-  return 0;
-}
-_EOF_
-$CC $CFLAGS -c conftest.c >/dev/null 2>/dev/null
-[ $? -ne 0 ] && CFLAGS="${CFLAGS} -DNO_ERRNO"
-
-
-echo Check for directory libraries
-cat > conftest.c << _EOF_
-int main() { return closedir(opendir(".")); }
-_EOF_
-
-$CC -o conftest conftest.c >/dev/null 2>/dev/null
-if [ $? -ne 0 ]; then
-  OPT=""
-  for lib in ndir dir ucb bsd BSD PW x dirent
-  do
-    $CC -o conftest conftest.c -l$lib >/dev/null 2>/dev/null
-    [ $? -eq 0 ] && OPT=-l$lib && break
-  done
-  if [ ${OPT} ]; then
-    LFLAGS2="${LFLAGS2} ${OPT}"
-  else
-    CFLAGS="${CFLAGS} -DNO_DIR"
-  fi
-fi
-
-
-# Dynix/ptx 1.3 needed this
-
-echo Check for readlink
-echo "int main(){ return readlink(); }" > conftest.c
-$CC -o conftest conftest.c >/dev/null 2>/dev/null
-if [ $? -ne 0 ]; then
-  $CC -o conftest conftest.c -lseq >/dev/null 2>/dev/null
-  [ $? -eq 0 ] && LFLAGS2="${LFLAGS2} -lseq"
-fi
-
-
 echo Check for directory include file
 OPT=""
 for inc in dirent.h sys/ndir.h ndir.h sys/dir.h
@@ -596,7 +517,6 @@
 done
 CFLAGS="${CFLAGS} ${OPT}"
 
-
 echo Check for nonexistent include files
 for inc in stdlib.h stddef.h unistd.h fcntl.h string.h
 do
