--- busybox-1.31.0/libbb/xconnect.c	2019-06-10 09:09:31.000000000 +0000
+++ src/libbb/xconnect.c	2019-08-09 20:30:05.933316235 +0000
@@ -11,6 +11,8 @@
 #include <netinet/in.h>
 #include <net/if.h>
 #include <sys/un.h>
+#include <netdb.h>
+#include "../libres/dietdns.h"
 #if ENABLE_IFPLUGD || ENABLE_FEATURE_MDEV_DAEMON || ENABLE_UEVENT
 # include <linux/netlink.h>
 #endif
@@ -285,7 +287,7 @@
 	 * for each possible socket type (tcp,udp,raw...): */
 	hint.ai_socktype = SOCK_STREAM;
 	hint.ai_flags = ai_flags & ~DIE_ON_ERROR;
-	rc = getaddrinfo(host, NULL, &hint, &result);
+	rc = diet_getaddrinfo(host, NULL, &hint, &result); /* hack to use dietlibc's resolver to work around static compile issue */
 	if (rc || !result) {
 		bb_error_msg("bad address '%s'", org_host);
 		if (ai_flags & DIE_ON_ERROR)
@@ -312,7 +314,7 @@
 	set_nport(&r->u.sa, htons(port));
  ret:
 	if (result)
-		freeaddrinfo(result);
+		diet_freeaddrinfo(result); /* hack to use dietlibc's resolver to work around static compile issue */
 	return r;
 }
 #if !ENABLE_FEATURE_IPV6
