--- ../networking/httpd.c	2019-06-10 09:09:31.000000000 +0000
+++ ./networking/httpd.c	2019-07-07 23:36:44.260599490 +0000
@@ -2208,7 +2208,7 @@
 		if (proxy_entry) {
 			if (verbose > 1)
 				bb_error_msg("proxy:%s", urlcopy);
-			lsa = host2sockaddr(proxy_entry->host_port, 80);
+			lsa = host2sockaddr(proxy_entry->host_port, 8080);
 			if (!lsa)
 				send_headers_and_exit(HTTP_INTERNAL_SERVER_ERROR);
 			proxy_fd = socket(lsa->u.sa.sa_family, SOCK_STREAM, 0);
