--- vivid-0.6.0/src/main.rs	2020-09-20 16:26:54.000000000 +0000
+++ ./src/main.rs	2020-09-22 10:57:48.808990524 +0000
@@ -11,7 +11,7 @@
 use std::process;
 
 use clap::{
-    crate_description, crate_name, crate_version, App, AppSettings, Arg, ArgMatches, SubCommand,
+    crate_description, crate_name, App, AppSettings, Arg, ArgMatches, SubCommand,
 };
 
 use crate::color::ColorMode;
@@ -47,7 +47,7 @@
     let database_path_env_s = env::var("VIVID_DATABASE").ok();
     let database_path_env = database_path_env_s.as_ref().map(Path::new);
 
-    let database_path_system = Path::new("/usr/share/vivid/filetypes.yml");
+    let database_path_system = Path::new("@MAGISK_PREFIX@/usr/share/vivid/filetypes.yml");
 
     let database_path = database_path_from_arg
         .or(database_path_env)
@@ -82,7 +82,7 @@
     theme_path_user.push(theme_file.clone());
 
     let mut theme_path_system = PathBuf::new();
-    theme_path_system.push("/usr/share/vivid/themes/");
+    theme_path_system.push("@MAGISK_PREFIX@/usr/share/vivid/themes/");
     theme_path_system.push(&theme_file);
 
     let theme_path =
@@ -103,7 +103,7 @@
 
 fn run() -> Result<()> {
     let app = App::new(crate_name!())
-        .version(crate_version!())
+        .version("@MAGISK_MODULE_VERSION@ - Compiled for Android by nelshh @ XDA")
         .about(crate_description!())
         .global_setting(AppSettings::ColorAuto)
         .global_setting(AppSettings::ColoredHelp)
