--- coreutils-8.32/configure	2020-03-05 13:24:44.000000000 +0000
+++ ./configure	2020-04-13 06:01:49.510092029 +0000
@@ -46107,22 +46107,10 @@
 
 
 
-  WANT_MKTIME_INTERNAL=0
-    ac_fn_c_check_func "$LINENO" "__mktime_internal" "ac_cv_func___mktime_internal"
-if test "x$ac_cv_func___mktime_internal" = xyes; then :
-
-$as_echo "#define mktime_internal __mktime_internal" >>confdefs.h
-
-
-else
-            WANT_MKTIME_INTERNAL=1
-
+WANT_MKTIME_INTERNAL=1
 $as_echo "#define NEED_MKTIME_INTERNAL 1" >>confdefs.h
 
 
-fi
-
-
   if test $WANT_MKTIME_INTERNAL = 1; then
 
 
@@ -46230,7 +46218,7 @@
 
 
   # Determine how to get the list of mounted file systems.
-  ac_list_mounted_fs=
+  ac_list_mounted_fs=found
 
   # If the getmntent function is available but not in the standard library,
   # make sure LIBS contains the appropriate -l option.
