--- ../help.c	2019-06-07 18:37:32.000000000 +0000
+++ ./help.c	2019-07-18 12:50:43.690541292 +0000
@@ -728,7 +728,8 @@
 	 *
 	 * Always show the version, even if other options are given.
 	 */
-	printf("git version %s\n", git_version_string);
+	printf("git - version %s\n", git_version_string);
+	printf("compiled for Android by nelshh @ xda-developers\n\n");
 
 	if (build_options) {
 		printf("cpu: %s\n", GIT_HOST_CPU);
