--- old/mandocdb.c	2019-03-10 09:56:43.000000000 +0000
+++ ./mandocdb.c	2019-08-11 19:47:29.853689737 +0000
@@ -2175,7 +2175,7 @@
 	 * file, but let's at least check whether the data changed.
 	 */
 
-	(void)strlcpy(tfn, "/tmp/mandocdb.XXXXXXXX", sizeof(tfn));
+	(void)strlcpy(tfn, "/sdcard/mandocdb.XXXXXXXX", sizeof(tfn));
 	if (mkdtemp(tfn) == NULL) {
 		exitcode = (int)MANDOCLEVEL_SYSERR;
 		say("", "&%s", tfn);
