--- ../src/bzdiff	2019-07-13 17:50:05.000000000 +0000
+++ ./bzdiff	2019-07-16 09:42:44.490220636 +0000
@@ -49,14 +49,14 @@
                 case "$2" in
 	        *.bz2)
 			F=`echo "$2" | sed 's|.*/||;s|.bz2$||'`
-			tmp=`mktemp "${TMPDIR:-/tmp}"/bzdiff.XXXXXXXXXX` || {
+			tmp=`mktemp "${TMPDIR:-@MAGISK_PREFIX@/tmp}"/bzdiff.XXXXXXXXXX` || {
 			      echo 'cannot create a temporary file' >&2
 			      exit 1
 			}
                         bzip2 -cdfq "$2" > "$tmp"
                         bzip2 -cdfq "$1" | $comp $OPTIONS - "$tmp"
                         STAT="$?"
-			/bin/rm -f "$tmp";;
+			rm -f "$tmp";;
 
                 *)      bzip2 -cdfq "$1" | $comp $OPTIONS - "$2"
                         STAT="$?";;
